draft/spec/index.html

```diff
@@ -1514,6 +1547,27 @@ <h2>Moab in an OCFL Object</h2>
 }
 </pre>
 </section>
+    <section id="example-extended-object">
+        <h2>Example Extended OCFL Object</h2>
+        <p>
+            The following example OCFL Object has an extension containing custom content. The OCFL Object itself
+            remains valid.
+        </p>
+<pre>
+[object root]
+    ├── 0=ocfl_object_1.0
+    ├── inventory.json
+    ├── inventory.json.sha512
+    ├── extensions
+    │   └── 0011-my-extension
```

Comment:
Do we need to create an example extension, maybe `9999-example-extension` or such in order for this to be valid?

Reply:
we have an example one `0000-example-extension` see: https://github.com/OCFL/extensions/blob/master/docs/0000-example-extension.md

Reply:
Thanks @rosy1280 , I'd forgotten that. @awoods change looks good so will resolve